arborist/server.go

```diff
@@ -673,14 +673,19 @@ func (server *Server) handleResourceCreate(w http.ResponseWriter, r *http.Reques
 		_ = errResponse.write(w, r)
 		return
 	}
-	created := resourceFromQuery.standardize()
-	server.logger.Info("created resource %s (%s)", created.Path, created.Tag)
-	result := struct {
-		Created *ResourceOut `json:"created"`
-	}{
-		Created: &created,
+	out := resourceFromQuery.standardize()
+	if errResponse != nil {
+		server.logger.Info("not creating resource %s (%s), already exists", out.Path, out.Tag)
+		_ = jsonResponseFrom(out, 409).write(w, r)
```

Comment:
It would be nice if we still have the same
```json
{"error": {"message": ..., "code": ...}, "exist": {"name": ..., "path": ..., "tag": ..., ...}}
```
error response structure in the body, but current response is just fine too.

Reply:
I like it, more consistent with the current pattern. I'll go with this.